js/index.js

```diff
@@ -0,0 +1,47 @@
+$(function(){
+
+  var total = 0;
+  $("#temperature").submit(handleSubmit);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    var enteredTemperature = $("#originalTemp").val();
+
+    $("#originalTemp").val("");
+
+    var parsedTemperature = parseFloat(enteredTemperature);
+
+    total = parsedTemperature;
+
+    total = total * 1.8 + 32
+
+    $("#result").text(total.toFixed(0) + "° F");
+
+    if (total < 40) {
+      $("body").attr("class", "cold");
+      $("#result").css("color", "#151B54");
```

Comment:
@chloeeffron nice touch to change the text color to accent the background image 💯